packages/vats/src/provisionPoolKit.js

```diff
@@ -95,13 +100,23 @@ export const prepareProvisionPoolKit = (
     'ProvisionPoolKit',
     {
       machine: M.interface('ProvisionPoolKit machine', {
-        makeHandler: M.call({
+        addRevivableAddresses: M.call(M.arrayOf(M.string())).returns(),
+        getWalletReviver: M.call().returns(
+          M.remotable('ProvisionPoolKit wallet reviver'),
+        ),
+        setReferences: M.call({
           bankManager: M.any(),
           namesByAddressAdmin: M.any(),
           walletFactory: M.any(),
-        }).returns(M.remotable('BridgeHandler')),
+        }).returns(),
```

Comment:
ci shows:
```
2023-05-04T01:49:50.431Z SwingSet: ls: v31: Error#1: value for bankManager is not durable: slot 0 of { body: '#"&0"', slots: [ 'p-72' ] }
...
at setReferences (.../vats/src/provisionPoolKit.js:181)
```

I think the '&' sigil on `&0` means there's a promise there. Perhaps the exo method guard for `setReferences` should use `M.callWhen()` and `M.await()`.